circle.yml

```diff
@@ -97,7 +97,7 @@ commands:
       - attach_workspace:
           at: ~/
       - run:
-          command: yarn lerna exec --scope @packages/server "yarn test-e2e --chunk << parameters.chunk >> --browser << parameters.browser >>"
+          command: yarn lerna exec --scope @packages/server "yarn test ./test/e2e/<< parameters.chunk >>*spec* --browser << parameters.browser >>"
```

Comment:
why the *spec* ?

Reply:
just to be extra specific... we name all our tests `N_something_spec.extension`. think it's unnecessary?